src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -214,6 +218,57 @@ public ITextComponent getShortDisplayString()
         }
     }
 
+    public static class SmeltAbleOreRequest extends AbstractRequest<SmeltableOre>
+    {
+
+        private static ImmutableList<ItemStack> oreExamples;
+
+        SmeltAbleOreRequest(@NotNull final IRequester requester, @NotNull final IToken token, @NotNull final SmeltableOre requested)
+        {
+            super(requester, token, requested);
+        }
+
+        SmeltAbleOreRequest(
+                @NotNull final IRequester requester,
+                @NotNull final IToken token,
+                @NotNull final RequestState state,
+                @NotNull final SmeltableOre requested)
+        {
+            super(requester, token, state, requested);
+
+        }
+
+        @NotNull
+        @Override
+        public ITextComponent getShortDisplayString()
+        {
+            return new TextComponentTranslation(TranslationConstants.COM_MINECOLONIES_REQUESTS_SMELTABLE_ORE);
+        }
+
+        @Override
+        public List<ItemStack> getDisplayStacks()
+        {
+            if (oreExamples == null)
+            {
+                oreExamples =
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2696)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Synchronize this lazy initialization of 'oreExamples' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2444)